src/components/SingleTermView/Variants/VariantsTable.jsx

```diff
@@ -0,0 +1,230 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {
+    Box, Table, TableBody,
+    TableCell, TableContainer, TableHead,
+    TableRow, TableSortLabel,
+    Paper, Chip, Typography, IconButton, Pagination, PaginationItem
+} from '@mui/material';
+import DownloadOutlinedIcon from '@mui/icons-material/DownloadOutlined';
+import ArrowOutwardOutlinedIcon from '@mui/icons-material/ArrowOutwardOutlined';
+import DoneIcon from '@mui/icons-material/Done';
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+import CloseOutlinedIcon from '@mui/icons-material/CloseOutlined';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import { vars } from '../../../theme/variables';
+
+const {gray100, gray600, gray700, gray900} = vars;
+
+function descendingComparator(a, b, orderBy) {
+    if (b[orderBy] < a[orderBy]) {
+        return -1;
+    }
+    if (b[orderBy] > a[orderBy]) {
+        return 1;
+    }
+    return 0;
+}
+
+function getComparator(order, orderBy) {
+    return order === 'desc'
+        ? (a, b) => descendingComparator(a, b, orderBy)
+        : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+// stableSort brings sort stability to non-modern browsers
+function stableSort(array, comparator) {
+    const stabilizedThis = array.map((el, index) => [el, index]);
+    stabilizedThis.sort((a, b) => {
+        const order = comparator(a[0], b[0]);
+        if (order !== 0) {
+            return order;
+        }
+        return a[1] - b[1];
+    });
+    return stabilizedThis.map((el) => el[0]);
+}
+
+const paperStyle = {
+    width: '100%',
+    border: '1px solid #EAECF0',
+    boxShadow: '0px 1px 2px rgba(16, 24, 40, 0.05)',
+    borderRadius: '0.75rem'
+};
+
+const descriptionTextStyle = {
+    color: gray600,
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    whiteSpace: 'nowrap',
+    maxWidth: '43.75rem'
+};
+
+const iconButtonStyle = {
+    padding: '0.625rem',
+    background: 'transparent',
+    height: '2.5rem',
+    '&:hover': {
+        background: gray100
+    }
+};
+
+const getChipColor = (status) => ({
+    Active: 'success',
+    Inactive: 'default',
+    Deleted: 'error'
+}[status]);
+
+const getChipIcon = (status) => ({
+    Active: <DoneIcon />,
+    Inactive: <CloseOutlinedIcon />,
+    Deleted: <DeleteOutlineOutlinedIcon />
+}[status]);
+
+function CustomTableHead(props) {
```

Comment:
hey @Aiga115 can we split this into a separated file?

Reply:
could you also please check this

1. hover over table cells doesnt have the same styling as figma 
2. here as shown on the video, cells width keep changing when making the sort action, also sorting action doesn't apply from the first click, i have to click once more in order to apply the sort
https://github.com/MetaCell/interlex/assets/32265731/6fefe238-3324-4756-bf8d-14276e2ebcc1

3. table box shadow is not as figma
4. font weight for Organization  column and for titles should be 500
5. icon is not as figma 
<img width="52" alt="Screenshot 2024-06-04 at 1 49 27 AM" src="https://github.com/MetaCell/interlex/assets/32265731/b02ec7e3-a289-47aa-9db8-52d814451f6b">

thanks